tests/integration/standard/test_query.py

```diff
@@ -590,21 +608,29 @@ def test_prepare_batch_statement_after_alter(self):
 
         values_to_insert = [(1, 2, 3), (2, 3, 4), (3, 4, 5), (4, 5, 6)]
 
-        # We query the three hosts in order (due to the ForcedHostSwitchPolicy)
+        # We query the three hosts in order (due to the ForcedHostIndexPolicy)
         # the first three queries will have to be repreapred and the rest should
         # work as normal batch prepared statements
         for i in range(10):
+            policy.set_host(i % 3)
```

Comment:
same question than above. can we do that cycling over the `metadata.all_hosts()` list and use `host`?